README.md

```diff
@@ -0,0 +1,119 @@
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
[OPTIONAL]
- [ ] Kindly remove all the commented-out descriptions. There are just descriptions of how to fill your README file professionally.

_**This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change**_

Reply:
> [OPTIONAL]
> 
>     * [ ]  Kindly remove all the commented-out descriptions. There are just descriptions of how to fill your README file professionally.
> 
> 
> _**This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change**_